pom.xml

```diff
@@ -45,7 +45,7 @@
     <jetty.version>11.0.9</jetty.version>
     <maven.compiler.arg>-Werror</maven.compiler.arg>
     <mycore.version>2022.05-SNAPSHOT</mycore.version>
-    <node.version>v12.16.3</node.version>
+    <node.version>v12.22.12</node.version>
```

Comment:
Does not belong directly to it, but this props are redefined in MIR, may we should use definitions of mycore-parent, according to https://github.com/MyCoRe-Org/mycore/pull/1623